pkg/operator/ceph/cluster/controller.go

```diff
@@ -1173,3 +1184,8 @@ func populateConfigOverrideConfigMap(context *clusterd.Context, namespace string
 
 	return nil
 }
+
+func hasCleanupPolicy(cephCluster *cephv1.CephCluster) bool {
+	policy := cephCluster.Spec.CleanupPolicy
+	return policy.DeleteDataDirOnHosts != ""
```

Comment:
Do we need to handle the case when user deletes the clean up policy ? For example, user adds cleanup policy (no orchestration will run in that case), but user then decides otherwise and updates the cluster to remove the clean up policy (orchestration will run this time!)

Reply:
Per separate discussion, if they remove the policy later, the cluster can start orchestrating again since the CR wasn't deleted yet.